client/me/billing-history/transactions-table.jsx

```diff
@@ -87,11 +88,17 @@ class TransactionsTable extends React.Component {
 
 	serviceNameDescription = transaction => {
 		let description;
+		const termLabel = getPlanTermLabel( transaction.wpcom_product_slug, this.props.translate );
```

Comment:
If `transaction.domain` is falsy, this `termLabel` computation will be in vain. I'd suggest us to move this under the truthy block of the conditional statement `if( transaction.domain )` below.